source/fe/fe_system.cc

```diff
@@ -1994,7 +1994,7 @@ FESystem<dim, spacedim>::get_subface_interpolation_matrix(
   FullMatrix<double> &                interpolation_matrix) const
 {
   AssertThrow(
-    (x_source_fe.get_name().find("FE_System<") == 0) ||
+    (x_source_fe.get_name().find("FESystem<") == 0) ||
```

Comment:
Oy, how did this work? What does `FESystem`'s `get_name()` function return?

Reply:
It returns the correct value. We are saved by the dynamic cast on the next line :)

Reply:
Ha, bizarre!